src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
```

Comment:
Consider using a more appropriate CSS property than `font-style` for `.UserInfo`. `font-style` is used to set the font style to italic or oblique. Consider using `font-weight` or `font-size` instead. 🎨